<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Description" content="Jin Woo Oh, Ph.D. – Computational Biologist | Gene Regulation &amp; Oncology">
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="2575.6">
  <style type="text/css">
    p.p2 {margin: 0.0px 0.0px 12.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    p.p3 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; color: #0000e9; -webkit-text-stroke: #0000e9}
    p.p5 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; color: #6d6d6d; -webkit-text-stroke: #6d6d6d; min-height: 14.0px}
    li.li7 {margin: 0.0px 0.0px 16.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    li.li8 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    span.s1 {font-kerning: none}
    span.s2 {text-decoration: underline ; font-kerning: none}
    span.s3 {font-kerning: none; color: #000000; -webkit-text-stroke: 0px #000000}
    span.s4 {text-decoration: underline ; font-kerning: none; -webkit-text-stroke: 0px #0000e9}
    span.s5 {-webkit-text-stroke: 0px #000000}
    span.s6 {font: 12.0px Times; text-decoration: underline ; font-kerning: none; color: #0000e9; -webkit-text-stroke: 0px #0000e9}
    ol.ol1 {list-style-type: decimal}
    ul.ul1 {list-style-type: disc}
  </style>
</head>
<body>
<h1 style="margin: 0.0px 0.0px 16.1px 0.0px; font: 24.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Jin Woo Oh, Ph.D.</b></span></h1>
<p class="p2"><span class="s1">Computational Biologist | Gene Regulation &amp; Oncology</span></p>
<p class="p3"><span class="s2">Email</span><span class="s3"> <a href="https://www.linkedin.com/in/jin-woo-oh-65113b90/"><span class="s4">LinkedIn</span></a> <a href="https://scholar.google.com/citations?user=V_mNtIQAAAAJ&amp;hl=en&amp;oi=ao"><span class="s4">Google Scholar</span></a></span></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>About Me</b></span></h2>
<p class="p2"><span class="s1">I develop advanced computational methods to uncover how biological systems and cellular processes operate, with the goal of identifying disease mechanisms that inform new therapeutic strategies and improve human health.</span></p>
<p class="p5"><span class="s1"></span><br></p>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Experience</b></span></h3>
<ul class="ul1">
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>Research Scientist, Lunit Inc.<br>
</b>Jun 2024 – Present Developing integrative AI models for cancer histopathology image analysis using spatial transcriptomics data to support translational oncology research initiatives.</span></li>
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>Graduate Student Researcher, Johns Hopkins University<br>
</b>Sep 2018 – May 2024 I developed computational methods to decipher the regulatory genome and collaborated to functionally characterize human and mouse genomes, aiming to advance the diagnosis and treatment of complex diseases such as cancer.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Education</b></span></h3>
<ul class="ul1">
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>Ph.D. in Biomedical Engineering, Johns Hopkins University </b><a href="https://jscholarship.library.jhu.edu/items/8fa3260a-ccc3-4957-b1f0-eb2c8b3de4ca"><span class="s6"><b>(Thesis)</b></span></a><b><br>
</b>2018 – 2024</span></li>
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>M.S. in Biomedical Engineering, Johns Hopkins University<br>
</b>2017 – 2018</span></li>
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>B.S. Biomedical Engineering &amp; Applied Mathematics and Statistics, Johns Hopkins University<br>
</b>2013 – 2017</span></li>
</ul>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Other Training</b></span></h2>
<ul class="ul1">
  <li class="li8"><span class="s5"></span><span class="s1">Consortium trainee, IGVF (Impact of Genomic Variation on Function), 2022–2024.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Consortium trainee, ENCODE (Encyclopedia of DNA Elements), 2018–2024.</span></li>
</ul>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Awards &amp; Honors</b></span></h2>
<ul class="ul1">
  <li class="li8"><span class="s5"></span><span class="s1">The Mette Strand Young Investigator Award, Johns Hopkins School of Medicine, 2024.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Reviewer’s Choice Top Abstract Award, American Society of Human Genetics, 2023.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">IGVF Consortium Poster Award, IGVF Consortium, 2022.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">ENCODE Consortium Team Science Award, ENCODE Consortium, 2022.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">David T. Yue Memorial Award for Teaching Excellence, Johns Hopkins University, 2017.</span></li>
</ul>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Publications</b></span></h2>
<ol class="ol1">
  <li class="li8"><span class="s5"></span><span class="s1">Jin Woo Oh &amp; Michael A Beer. “Gapped-kmer sequence modeling robustly identifies regulatory vocabularies and distal enhancers conserved between evolutionarily distant mammals.” <i>Nature Communications</i> (2024)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Jin Woo Oh*, David Yao*, Josh Tycko*, Lexi R Bounds*, Sager J Gosai*, Lazaros Lataniotis*, Ava Mackay Smith, Benjamin R Doughty, Idan Gabdank, Henri Schmidt, Ingrid Youngworth, Kalina Andreeva, Xingjie Ren, Alejandro Barrera, Yunahi Luo, Keith Siklenka, Galip Gurkan Yardimci, Ryan Tewhey, Anshul Kundaje, William J Greenleaf, Pardis C Sabeti, Christina Leslie, Yuri Pritykin, Jill E Moore, Michael A Beer, Charles Gersbach, Timothy E Reddy, Yin Shen, Jesse M Engreitz, Michael C Bassik, Steven K Reilly. “Multicenter integrated analysis of noncoding CRISPRi screens.” <i>Nature Methods</i> (2024)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Renhe Luo, Jielin Yan, Jin Woo Oh, Wang Xi, Dustin Shigaki, Wilfred Wong, Hyein S Cho, Dylan Murphy, Ronald Cutler, Bess P Rosen, Julian Pulecio, Dapeng Yang, Rachel A Glenn, Tingxu Chen, Qing V Li, Thomas Vierbuchen, Simone Sidoli, Effie Apostolou, Danwei Huangfu, Michael A Beer. “Dynamic network-guided CRISPRi screen identifies CTCF-loop-constrained nonlinear enhancer gene regulatory activity during cell state transitions.” <i>Nature Genetics</i> (2023)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">IGVF. “Deciphering the impact of genomic variation on function.” <i>Nature</i> (2024)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Jin Woo Oh. “Enhanced Algorithms to detect and characterize conserved regulatory sequences.” <i>Johns Hopkins University</i> (2024)</span></li>
</ol>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Published Abstracts</b></span></h2>
<ol class="ol1">
  <li class="li8"><span class="s5"></span><span class="s1">Jin Woo Oh, Sanghoon Song, Soohyun Hwang, Mingu Kang, Jinhee Lee, Sergio Pereira, Hyunsu Kim, Sehhoon Park, Junhun Cho, Se-Hoon Lee, Han-Sin Jeong, Chang Ho Ahn, Siraj M Ali, Chan-Young Ock, Myung-Ju Ahn. “A novel single-cell level approach integrating artificial intelligence (AI)–powered histomorphology labeling and spatial transcriptomics enables biomarker identification of treatment resistance in salivary gland cancer (SGC).” Proceedings of the AACR (2025)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Hyunsu Kim, Sehhoon Park, Jin Woo Oh, Soohyun Hwang, Jinyoung Kim, Eun-hye Kim, Nayeon Choi, Junhun Cho, Hyun-Ae Jung, Dongryul Oh, Se-Hoon Lee, Yong Chan Ahn, Han-Sin Jeong, Chang Ho Ahn, Chan-Young Ock, Myung-Ju Ahn. “Exploratory analysis of tumor microenvironment using scRNA, scTCR, and spatial transcriptomics in salivary gland cancer with surgical sample after neoadjuvant immuno-chemotherapy.” Proceedings of the AACR (2025)</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">Seungeun Lee, Jin Woo Oh, Soohyun Hwang, Jeanne Shen, Sehhoon Park, Hyojin Kim, Young Kwang Chae, Se-Hoon Lee, Yoon-La Choi, Jin-haeng Chung, Jaewoong Shin, Heon Song, Aaron Valero Puche, Donggeun Yoo, Taebum Lee, Chiyoon Oum, Jeongmi Kim, Siraj M Ali, Chan-Young Ock. “Deep learning–powered H&amp;E whole-slide image analysis of endothelial cells to characterize tumor vascular environment and correlate treatment outcome to immunotherapy.” <i>Journal of Clinical Oncology</i> (2025)</span></li>
</ol>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Poster Presentations</b></span></h2>
<ul class="ul1">
  <li class="li8"><span class="s5"></span><span class="s1">“A novel single-cell level approach integrating AI–powered histomorphology labeling and spatial transcriptomics enables biomarker identification of treatment resistance in salivary gland cancer (SGC).” AACR, 2025.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">“Functional characterization of the mammalian genome.” ASHG, 2023.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">“Gapped-kmer based machine learning and sequence alignment identify conserved regulatory vocabularies and enhancers in evolutionarily distant mammals.” IGVF Consortium Meeting, 2022.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">“Identifying subsets of cell-specific regulatory sequence features from heterogeneous tissue-specific chromatin state data using predictive gapped k-mer features.” Cold Spring Harbor Lab – Biology of Genomes, 2019.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">“Identifying subsets of cell-specific regulatory sequence features from heterogeneous tissue-specific chromatin state data using predictive gapped k-mer features.” ENCODE Consortium Meeting, 2019.</span></li>
</ul>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Invited Talks</b></span></h2>
<ul class="ul1">
  <li class="li8"><span class="s5"></span><span class="s1">“Multi-center integrative analysis of non-coding CRISPR screens.” ENCODE Consortium Meeting, 2022.</span></li>
  <li class="li8"><span class="s5"></span><span class="s1">“Sequence alignment using gapped-kmer features identifies conserved orthologous mammalian enhancers with high precision.” ENCODE Consortium Meeting, 2021.</span></li>
</ul>
<p class="p5"><span class="s1"></span><br></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Leadership &amp; Service</b></span></h2>
<ul class="ul1">
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>Review Editor, Frontiers in Epigenetics and Epigenomics<br>
</b>Nov 2024 – Present</span></li>
  <li class="li7"><span class="s5"><b></b></span><span class="s1"><b>President, Korean Graduate Student Association (KGSA)<br>
</b>Sep 2019 – Jun 2020, Johns Hopkins University</span></li>
</ul>
</body>
</html>
